source/guides/getting-started.md

```diff
@@ -713,15 +713,16 @@ require 'spec_helper'
 require_relative '../../../../apps/web/controllers/books/create'
 
 describe Web::Controllers::Books::Create do
+  let(:action) { Web::Controllers::Books::Create.new }
   let(:params) { Hash[book: { title: 'Confident Ruby', author: 'Avdi Grimm' }] }
 
-  after do
+  before do
     BookRepository.clear
   end
 
   it 'creates a new book' do
     action.call(params)
-    action.book.id.wont_be_nil
+    BookRepository.all.count.wont_equal 0
```

Comment:
So I don't really know how to avoid `BookRepository` call in this test. What would be the "lotus" way? Or at least the cleanest OOP way of doing this?